src/sentry/static/sentry/app/views/organizationEventsV2/eventModalContent.jsx

```diff
@@ -50,7 +51,11 @@ const EventModalContent = props => {
           })}
       </HeaderBox>
       <ContentColumn>
-        <EventInterfaces event={event} projectId={projectId} />
+        {event.type === 'transaction' ? (
+          <TransanctionView event={event} />
+        ) : (
+          <EventInterfaces event={event} projectId={projectId} />
```

Comment:
Shouldn't an event have a projectId property?

Reply:
@lynnagara I'm unsure, the prop was already there. I won't be able to refactor `<EventInterfaces/>` in this PR.